tests/ci/azure_pipeline_test/dsvm_nightly_linux_cpu_master.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+schedules:
```

Comment:
Right now the ci folder is really crowded, I think we should make it simpler (not sure if here or in a different PR). Maybe we should have a folder for the tests with normal ADO and another one for tests through AzureML. We might want to have the same structure in other BP repos